glib-macros/tests/props.rs

```diff
@@ -0,0 +1,57 @@
+use glib::prelude::*;
+#[test]
+fn props() {
+    mod foo {
+        use glib::prelude::*;
+        use glib::subclass::prelude::*;
+        use glib_macros::Props;
+        use std::cell::RefCell;
+
+        mod imp {
+            use super::*;
+
+            #[derive(Props, Default)]
+            pub struct Foo {
+                #[prop("bar", "bar", "This is a bar", None, glib::ParamFlags::READWRITE)]
```

Comment:
I think those should be named attributes and they should have sane defaults if/when possible. Something to note is in general people would want to use a specific function for the setters, so there should be some way to set that.

Reply:
Once they are named attributes, you could make the first three optional and use the field name as the default value

Reply:
Makes sense, I'll do it